Tests/UITests/TestApp/ViewsTests/MarkdownDocumentViewTest.swift

```diff
@@ -0,0 +1,116 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+// swiftlint:disable file_types_order
+
+import SpeziFoundation
+import SpeziViews
+import SwiftUI
+
+
+struct MarkdownDocumentViewTest: View {
+    private let document = try! MarkdownDocument( // swiftlint:disable:this force_try
+        processing: """
+            ---
+            title: Welcome to the Spezi Ecosystem
+            date: 2025-06-22T14:41:16+02:00
+            ---
+            
+            # Welcome to the Spezi Ecosystem
+            This article aims to provide you with a broad overview of Spezi.
+            
+            <marquee />
```

Comment:
Do we want to provide a test where we also parse some arguments, might make this a bit more expressive and we can demonstrate this in the code example.

Reply:
updated the test to specify multiple parameters of the custom view (image&period) via the element attrs